p2p/node/p2p_services.go

```diff
@@ -102,10 +103,18 @@ func (p *P2PNode) requestFromPeer(peerID peer.ID, topic *pubsubManager.Topic, re
 			// Finally, BlockView and HeaderView have different hash functions
 			case *types.WorkObjectBlockView:
 				if reqData == recvdType.Hash() {
+					// If gas limit if the older than the hash rate after the fork
```

Comment:
what? Something got lost in translation, haha

Reply:
not sure what hash rate has to do with anything here

Reply:
oops, correcting the comment now